app/components/domains-table.tsx

```diff
@@ -89,6 +90,25 @@ export default function DomainsTable(props: DomainsTableProps) {
     }
   }
 
+  function renderDnsRecordName(domainName: string) {
+    const words: string[] = domainName.split('.');
+    const nameBase: string = words[0];
+    const restOfName: string = words.slice(1).join('.');
+
+    return (
+      <Flex alignItems="flex-end" flexDirection="row">
+        <Text>
+          <Text as="span" sx={{ fontSize: 'md', fontWeight: 'medium' }}>
```

Comment:
I wonder if we should only do the bold/gray change, but leave the size the same?  I think it looks a bit off to have the name be larger.

Reply:
I don't mind making it the same font size